editor/src/core/shared/github/helpers.ts

```diff
@@ -305,36 +325,38 @@ function updateUserDetailsFromServerActions(userDetails: GithubUser): EditorActi
   return [updateGithubData({ githubUserDetails: userDetails })]
 }
 
-export async function getUserDetailsFromServer(): Promise<GithubUser | null> {
-  const url = GithubEndpoints.userDetails()
+export const GithubHelpers = {
```

Comment:
what's the reason for moving the function into this const? is it for easier stubbing of the method in the tests? (in `github-operations.test-utils.ts`) if that's the case, wouldn't it be easier to just leave it as a function (since it's already inside a helpers file) and stub it with a `*` import?

Reply:
It's exactly for mocking, I tried other approaches but right now the best price-to-value option is exporting through a const